### ISSUE TITLE
Errors and Server Responses Doc Update 

### ISSUE THREAD

**Issue by tinefin** [pantheon-systems/documentation#3164]
Re: [Errors and Server Responses](https://pantheon.io/docs/errors-and-server-responses/)

Priority (Low‚ Medium‚ High):

## Issue Description:

On Pantheon, we have a custom 561 http status code that will be returned instead of 403 on "directory listing denied". Would be good to add to this doc.

Reference: 96312 + BUGS-1752

## Suggested Resolution

**Comment by alexfornuto** [comment]
@tinefin it doesn't _always_ mean 403, right? As in, there are other situations that result in seeing a 561 page, aren't there?

**Comment by tinefin** [comment]
No, 561 is mainly for "561 no site installed" (fairly rare scenario).
Pantheon also returns it instead of 403 on "directory listing denied" though (highly frequent scenario).

**Comment by tinefin** [comment]
Meaning - it will be returned both for "no site installed" (which is correct), but also for "403 directory listing denied" (see internal bug card above).

**Comment by alexfornuto** [comment]
@tinefin please take a look at #3287 and see if that addresses your issue fully.
